src/link.c

```diff
@@ -733,24 +742,50 @@ LinkTable *LinkTable_disk_open(const char *dirn)
 
 LinkTable *path_to_Link_LinkTable_new(const char *path)
 {
-    Link *link = path_to_Link(path);
-    LinkTable *next_table = link->next_table;
+    struct Link *link = NULL, *tmp_link = NULL;
```

Comment:
Comma operator should not be used. 
https://rules.sonarsource.com/c/RSPEC-878/

Reply:
Also I have typedef-ed Link, you didn't need to use ``struct``

Reply:
You have already merged these changes. Are you going to update your repo or are you wanting me to create another PR?

Reply:
I am just going to update the changes myself. I have just found myself some time to work on this project.